### ISSUE TITLE
Allow to copy the title of a post

### ISSUE THREAD

**Issue by sallyFoster** [aeharding/voyager#731]
Can you please allow us to copy the title of a post?

**Comment by sallyFoster** [comment]
@aeharding ?

**Comment by aeharding** [comment]
Yeah - this should probably be in the "Select Text" dialog like Apollo.

**Comment by sallyFoster** [comment]
@aeharding Can we get an option to also just copy the entire thing? Sometimes I just want to copy everything without having to be selecting everything by hand.

**Comment by sallyFoster** [comment]
@aeharding Did you release the fix on the update because I am not getting the option to copy the title or to pre-select all text.

**Comment by aeharding** [comment]
No, it will be in the next release

**Comment by sallyFoster** [comment]
@aeharding Now, I saw that the title gets added after tapping on "Select Text"; however, there is still no option to automatically copy all text 😞

**Comment by aeharding** [comment]
> @aeharding Now, I saw that the title gets added after tapping on "Select Text"; however, there is still no option to automatically copy all text 😞

Would an option to `select all` or or `copy all` be more helpful?

**Comment by sallyFoster** [comment]
@aeharding Yep, a "Copy all" button would be great! Also, I don't know if I am pushing my luck asking for this, but if you could also have 2 extra buttons saying "Copy Title" and "Copy body" that would be wonderful! Many times user want to copy everything or just the title or just the body, and having a quick and easy way to do these would be divine!
